### ISSUE TITLE
Feature request: Use CharSequence instead of String in setTitle and setContentText (and underlying classes)

### ISSUE THREAD

**Issue by melker** [mreram/ShowCaseView#8]
To me it seems that these are just forwarded to TextView.setText(...) and those support CharSequence
This way we may use SpannableStringBuilder to add text with colors/images/etc.

**Comment by tekinumut** [comment]
Yes same problem here. I think view is not related with the navigation bar. If you work on fullscreen there is no problem because there is no navigation bar. If not, then i see something like that. 

![navigationBarIssue](https://user-images.githubusercontent.com/33953921/73177519-4c1dc600-4120-11ea-8ee5-c6ba23343e1d.png)
